draft-irtf-cfrg-opaque.md

```diff
@@ -1489,7 +1489,7 @@ Output:
 - session_key, the shared session secret.
 
 Exceptions:
-- ServerAuthenticationMACError , the handshake fails.
+- ServerAuthenticationError , the handshake fails.
```

Comment:
I thought we agreed on your suggestion here
https://github.com/cfrg/draft-irtf-cfrg-opaque/pull/314#discussion_r798821465

Reply:
Upon rereading things and making this PR I fell back to my original suggestion. I don't think it's important that the application know it's the MAC that failed here, and it's only guidance anyway. (An implementation could choose to be more specific.)